saleor/plugins/webhook/tests/test_shipping_webhook.py

```diff
@@ -454,6 +457,97 @@ def test_order_shipping_methods(
     assert shipping_methods[0]["message"] == webhook_reason
 
 
+@mock.patch(
+    "saleor.plugins.webhook.plugin.WebhookPlugin.excluded_shipping_methods_for_order"
+)
+def test_draft_order_shipping_methods(
+    mocked_webhook,
+    staff_api_client,
+    order_with_lines,
+    permission_group_manage_orders,
+    settings,
+):
+    # given
+    order_with_lines.status = OrderStatus.DRAFT
+    order_with_lines.save(update_fields=["status"])
+    settings.PLUGINS = ["saleor.plugins.webhook.plugin.WebhookPlugin"]
+    webhook_reason = "spanish-inquisition"
+    excluded_shipping_method_id = order_with_lines.shipping_method.id
+    mocked_webhook.return_value = [
+        ExcludedShippingMethod(excluded_shipping_method_id, webhook_reason)
+    ]
+    permission_group_manage_orders.user_set.add(staff_api_client.user)
+    query = """
+      query DraftOrdersQuery {
+       draftOrders(first: 1) {
+         edges {
+           node {
+             shippingMethods {
+               id
+               name
+               active
+               message
+             }
+           }
+         }
+       }
+      }
+    """
```

Comment:
Cannot you use `ORDER_QUERY_SHIPPING_METHOD`?

Reply:
No, as `ORDER_QUERY_SHIPPING_METHOD` uses `orders` field, which doesn't return `DRAFT` orders